pmg/views.py

```diff
@@ -345,6 +346,18 @@ def committee_detail(committee_id):
     )
 
 
+@app.route('/committee/<int:committee_id>/follow-cte')
+@app.route('/committee/<int:committee_id>/follow-cte/')
+def committee_detail_follow_cte(committee_id):
+    follow_committee(committee_id)
+    
+    # Remove confirmation email flash
+    session['_flashes'] = []
+    flash('You have successfully followed this committee to receive email alerts.', 'success')
```

Comment:
How about "You're now following this committee and we'll send you email alerts when new content is posted."

Reply:
Yes, that's much better.